CHANGELOG.md

```diff
@@ -92,6 +92,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Include order line total price to webhook payload - #6354 by @korycins
 - Fix for fulfilling an order when product quantity equals allocated quantity - #6333 by @GrzegorzDerdak
 - Fix for the ability to filter products on collection - #6363 by @GrzegorzDerdak
+- Fix password reset request - #6351 by @Manfred-Madelaine-pro, Ambroise and Pierre
```

Comment:
@Manfred-Madelaine-pro Could you move this entry to the "Unreleased" section at the top? Now it's added in "2.11" which was already released, while your changes would be released in the next stable version 3.0.

Reply:
Okay !